node/src/tasks/bsp_charge_fees.rs

```diff
@@ -51,6 +83,68 @@ where
             storage_hub_handler,
         }
     }
+
+    async fn remove_file_from_forest(&self, file_key: &H256) -> anyhow::Result<()> {
```

Comment:
I'd move these implementations to the bottom of the file. That way it is easier to check what events this file reacts to, and how it handles them.

Reply:
Changed!